lib/src/ffmpeg/filters/audio_mix_filter.dart

```diff
@@ -2,18 +2,19 @@ import 'package:ffmpeg_cli/src/ffmpeg/ffmpeg_command.dart';
 
 /// Mixes multiple audio streams together into one.
 ///
-/// I think this will automatically cut the volume of each input proportional
+/// This will automatically cut the volume of each input proportional
 /// to the number of inputs, e.g., 1/2 volume for 2 inputs, 1/3 volume for 3
 /// inputs.
 class AMixFilter implements Filter {
   const AMixFilter({
-    required this.inputCount,
+    this.inputCount,
   });
 
-  final int inputCount;
+  /// Number of inputs (defaults to 2)
+  final int? inputCount;
 
   @override
   String toCli() {
-    return 'amix=inputs=$inputCount';
+    return 'amix${(inputCount != null) ? "=inputs=$inputCount" : ""}';
```

Comment:
With the conditional, this has now become quite difficult to read. Let's switch a ternary for clarity:

```dart
return inputCount != null //
  ? 'amix=inputs=${inputCount!}'
  : 'amix';
```

Let's adjust any similarly impacted methods.